whisper/whisperv5/whisper.go

```diff
@@ -377,7 +393,7 @@ func (w *Whisper) AddSymKeyFromPassword(password string) (string, error) {
 
 	// double check is necessary, because deriveKeyMaterial() is very slow
 	if w.symKeys[id] != nil {
-		return "", fmt.Errorf("critical error: failed to generate unique ID")
```

Comment:
Good question @ebuchman, I'll leave that to the Ethereum maintainers to answer as I don't have much context about the entire function.